index.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(source) {
 		}
 		throw new Error(err);
 	}
+	if (options && options.useDefaultInES6Modules) {
+		result.js = 'var _require = __webpack_require__\n\n  __webpack_require__ = function(mid){\n    var module = _require(mid)\n\n    return (module && module.__esModule && module.default) ? module.default : module\n  }\n' + result.js
```

Comment:
no options && `export default ${result.js}` or do I miss something here 😛 ? This is a breaking change and will be part of `v1.0.0` in the near future